lib/common.sh

```diff
@@ -289,6 +292,14 @@ elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
   export IRONIC_TAG="v24.1.1"
   export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
   export BMOBRANCH="${BMORELEASEBRANCH:-release-0.6}"
+elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.8" ]]; then
+  export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.8"}
+  export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.8"}
+  export BARE_METAL_OPERATOR_TAG="v0.7.0"
+  export KEEPALIVED_TAG="v0.7.0"
+  export IRONIC_TAG="v25.0.0"
+  export BMOCOMMIT="eea41f4fbcf73c5aae7e015ad552b137ae507dbb" # tag v0.6.1
```

Comment:
this makes little sense to me to point a v0.6.1 commit but BMO tag is v0.7.0

Reply:
Ah sorry. I didn't notice the comment :sweat_smile: Changing.